inst/rmarkdown/templates/template_sfp/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,109 @@
+---
+title: {{{title}}}
+subtitle: {{{subtitle}}}
+author: {{{authors}}}
+date: {{{date}}}
+output: 
+  bookdown::html_document2:
+    toc: true
+    toc_float: true
+    number_sections: true
+---
+
+<!--
+Instructies bij opmaak van veldprotocol:
+
+Hieronder vind je de verschillende onderdelen van een veldprotocol en een beschrijving wat er bij elk onderdeel vermeld moet worden. 
+Niet alle onderdelen zijn altijd van toepassing. Laat in die gevallen de titelhoofdingen staan en vermeld expliciet “niet van toepassing”.
+Probeer het veldprotocol generiek te maken in de mate van mogelijk en probeer te vermijden dat het veldprotocol enkel specifiek voor een bepaald project bruikbaar is. 
+Je kan het protocol generiek houden door bepaalde variabelen, zoals bijvoorbeeld de grootte van een proefvlak of de diepte van een bodembemonstering, niet vast te leggen in het protocol. 
+In bijlage kan je dan een tabel toevoegen waarin je per project aangeeft welke waardes gekozen moeten worden voor die variabelen. 
```

Comment:
I believe the ultimate result for project protocols would be that those values are not to be manually looked up in an appendix of each relevant thematic protocol, but specified as a Rmarkdown [parameter](https://rmarkdown.rstudio.com/lesson-6.html) by the project protocol Rmd (using the first value as default), i.e. by using `params` in YAML. Then, a project could uptake a thematic protocol and define its own project-specific parameters. Available parameter values should be checked for by restrictions in the thematic protocol; not sure how that goes.

Reply:
I opened an issue for this. See #7